src/logging.cc

```diff
@@ -4,9 +4,9 @@
  * \brief Implementation of loggers.
  * \author Tianqi Chen
  */
+#include <rabit/rabit.h>
```

Comment:
So you are replacing `common/sync.h` with `rabit/rabit.h`?

Reply:
This is all the code contained in original commom/sync.h:
```
#ifndef XGBOOST_COMMON_SYNC_H_
#define XGBOOST_COMMON_SYNC_H_
#include <rabit/rabit.h>
#endif  // XGBOOST_COMMON_SYNC_H_
```
Is there any reason to keep it?  If so I will revert this change and add a note to the code.

Reply:
Let's get rid of it then.